docs/installation_guide/installation.rst

```diff
@@ -120,6 +120,60 @@ the installation and accept every license agreement run ``./install --acceptlice
 
     --------------------------------------------------------------------------
 
+Selecting singer connectors
+'''''''''''''''''''''''''''
+
+You can install PipelineWise only with required connectors by using the `--connectors=` argument. For example if you
+need to replicate data only from MySQL and PostgreSQL into a Snowflake database you can install PipelineWise by
+running:
+
+.. code-block:: bash
+
+    $ ./install --connectors=tap-mysql,tap-postgres,target-snowflake
+
+Here’s the list of the singer connectors and if they are installed by default or not:
+
++----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
+| **Connector**              | **Install Command**                     | **Included in default install?** | **Note**                              |
++----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
+| all                        | ./install --connectors=all              |                                  | Installs every supported connector    |
++----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
+| tap-adwords                | ./install --connectors=tap-adwords      |                                  |                                       |
```

Comment:
Why there is no value for some taps in column "Included in default install?" ?

Reply:
Should it be "NO"? :)

Reply:
I left it empty for better visibility but it's not trivial so fixed it by 89f08c7ecd5a6de1bec5cd27e3d0ce5d4f91507e

thanks